contrib/libvirt/files/sysusers.conf

```diff
@@ -0,0 +1,6 @@
+# Create libvirt-qemu system user and libvirt group
+
+u virtqemu - "virtqemu user" /var/lib/libvirt /usr/bin/nologin
+m virtqemu kvm
```

Comment:
should probably also belong to disk

Reply:
this was mostly just copied from arch's sysusers file

Reply:
renamed to _libvirt-qemu as per the default for libvirt